src/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeSystem/ModuleDefinition.cs

```diff
@@ -55,5 +55,10 @@ public GuidHandle BaseGenerationId
                 return _reader.ModuleTable.GetEncBaseId();
             }
         }
+
+        public CustomAttributeHandleCollection GetCustomAttributes()
+        {
+            return _reader.CustomAttributes;
```

Comment:
This is incorrect. It is going to return all custom attributes in the custom attribute table, not just those parented by the module definition. The code here should mirror the equivalent method on AssemblyDefinition, with EntityHandle.ModuleDefinition replacing EntityHandle.AssemblyDefinition.